src/common/config.rs

```diff
@@ -160,6 +167,22 @@ impl Wallet713Config {
             }
         }
     }
+
+    pub fn owner_api_address(&self) -> String {
+        self.owner_api_address.as_ref().map(|a| a.clone()).unwrap_or_else(|| String::from("127.0.0.1:12233"))
```

Comment:
Maybe we should use the Grin port numbers as a default, and have a `1` in front for floonet

Reply:
thx!